src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -111,6 +112,15 @@ private static float getContributionVariance(List<AuthorDailyContribution> contr
                 addDailyContributionForNewDate(authorDailyContributions, commitStartDate);
             }
 
+
+            List<CommitResult> currentDayCommitResults =
+                    authorDailyContributions.get(authorDailyContributions.size() - 1).getCommitResults();
+            Predicate<CommitResult> commitResultDuplicatePredicate = currentDayCommitResult ->
+                    currentDayCommitResult.getHash().equals(commitResult.getHash());
```

Comment:
Could we use the `equals` method of `CommitResult` for this duplicate check?

Reply:
Hmm, issue is that two commits can have the same hash but different commit titles ( possibly other fields as well ).
The example command given in #927 demonstrates this
`./gradlew run -Dargs="--repos http://github.com/AY1920S1-CS2103T-T12-2/main.git --view --since 25/09/2019 --until 25/09/2019"`